src/main/java/nps/id/publicapi/java/client/connection/WebSocketConnector.java

```diff
@@ -81,6 +82,7 @@ public void connect() {
             var connectionHeaders = StompMessageFactory.connectionHeaders(authToken, webSocketOptions.getHeartbeatOutgoingInterval());
             var completableFuture = webSocketStompClient.connectAsync(uri, null, connectionHeaders, new StompSessionHandlerAdapterImpl());
             stompSession = completableFuture.get(10, TimeUnit.SECONDS);
+            currentToken = ssoService.getCurrentAuthToken();
```

Comment:
Since we store currentToken in WebSocketConnector instance (for reason of course) I think we can remove **currentAuthToken** field in SsoService, as its no longer used.